File: benchmark/results/java-invocation-script/run-004/b-baseline/output/com/metaengine/demo/analytics/services/EventRepository.java
package com.metaengine.demo.analytics.services;

import com.metaengine.demo.analytics.aggregates.Event;
import java.util.List;

/** EventRepository persistence service in the analytics domain. */
public class EventRepository {
    /** Persist a new Event from a partial input. */
    public Event create(Event input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Event by id, or null if not found. */
    public Event findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Events up to limit. */
    public List<Event> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Event by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/ts-script-sonnet/run-001/b-baseline/output/src/domain/analytics/services/EventRepository.ts
import { Event } from '../aggregates/Event';

/** EventRepository service. */
export class EventRepository {
  create(input: Partial<Event>): Event { throw new Error('not implemented'); }
  findById(id: string): Event | null { throw new Error('not implemented'); }
  list(limit: number): Event[] { throw new Error('not implemented'); }
  delete(id: string): void { throw new Error('not implemented'); }
}
